public/app.prod.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Noticed this doesn't have the `devtools_port` property (like the dev version) - does omitting this disable dev tools or does it revert to a default value (or something else)?